package.json

```diff
@@ -52,7 +52,7 @@
         "axios": "~1.2.0",
         "bcrypt": "~5.1.0",
         "body-parser": "~1.20.1",
-        "cheerio": "~1.0.0-rc.12",
+        "cheerio": "^1.0.0-rc.12",
```

Comment:
are we sure we want to use ^ in packages' versions?

Reply:
nope my mistake